evergreen/evergreen.yml

```diff
@@ -240,7 +240,16 @@ functions:
           export FLE_AWS_SECRET_ACCESS_KEY=${FLE_AWS_SECRET_ACCESS_KEY}
           ${PREPARE_SHELL}
           SSL=${SSL} evergreen/add-certs-if-needed.sh
-          AUTH=${AUTH} SSL=${SSL} MONGODB_URI="${MONGODB_URI}" TOPOLOGY=${TOPOLOGY} OS=${OS} COMPRESSOR=${COMPRESSOR} evergreen/run-tests.sh
+          CLIENT_PEM=${DRIVERS_TOOLS}/.evergreen/x509gen/client.pem source evergreen/convert-client-cert-to-pkcs12.sh
```

Comment:
We need to source the file so that the export statements in the script end up being executed in the context of the calling shell (i.e. so that the variable exported by the shell script end up being set in the calling environment).

Reply:
Turns out this doesn't work because the `export` statements aren't working as expected after running `source`. EG logs revealed that they were not properly being exported. So, instead, we move this logic into `run-tests.sh`